<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>370</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>241</width>
    <height>370</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>241</width>
    <height>370</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>241</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Label at top) Label Stylesheet */

QLabel {
  qproperty-alignment: 'AlignVCenter | AlignRight';
  border: 1px solid gray;
}

background-color : white;

/* End Label Stylesheet */</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClear">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Clear,+/-,%,. buttons) Unary Operations Buttons Stylesheet */

QPushButton {
  background-color: rgb(58, 58, 58);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
/* End Unary Operations Buttons Stylesheet */
</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonplusMinus">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Clear,+/-,%,. buttons) Unary Operations Buttons Stylesheet */

QPushButton {
  background-color: rgb(58, 58, 58);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
/* End Unary Operations Buttons Stylesheet */
</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDivide">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Orange buttons) Operations Buttons Stylesheet */
QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: rgb(96, 96, 96);
	border:3px solid black;
}
.activeOperator {
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string>รท</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPercent">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Clear,+/-,%,. buttons) Unary Operations Buttons Stylesheet */

QPushButton {
  background-color: rgb(58, 58, 58);
  border: 1px solid black;
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #BEBEBE, stop: 1 #D7D7D7);
}
/* End Unary Operations Buttons Stylesheet */
</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num9">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num8">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num7">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonMultiply">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Orange buttons) Operations Buttons Stylesheet */
QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: rgb(96, 96, 96);
	border:3px solid black;
}
.activeOperator {
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num6">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num5">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSubtract">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>190</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Orange buttons) Operations Buttons Stylesheet */
QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: rgb(96, 96, 96);
	border:3px solid black;
}
.activeOperator {
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAdd">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>250</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Orange buttons) Operations Buttons Stylesheet */
QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: rgb(96, 96, 96);
	border:3px solid black;
}
.activeOperator {
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonEquals">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* (Orange buttons) Operations Buttons Stylesheet */
QPushButton {
  background-color: rgb(255, 151, 57);
  color: white; 
  border: 1px solid black;
}

QPushButton:pressed {
    background-color: rgb(96, 96, 96);
	border:3px solid black;
}
.activeOperator {
	border:3px solid black;
}</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_num0">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>310</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_decimal">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>61</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Digits 0-9 Stylesheet */
QPushButton {
   border: 1px solid black;
	
	background-color: rgb(92, 92, 92);
}
QPushButton:pressed {
    background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #dadbde, stop: 1 #f6f7fa);
}

/* End Digits 0-9 Stylesheet */</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
